src/layer/BingTiledImageLayer.js

```diff
@@ -61,14 +61,35 @@ define([
 
             this.attributionImage = WorldWind.configuration.baseUrl + "images/powered-by-bing.png";
 
-            this.attribution = this.createBingLogotype();
+            /**
+             * An {@link Offset} indicating where to place the Bing logo on the screen.
+             * @type {Offset}
+             * @default The lower right corner of the window.
+             */
+            this.logoPlacement = new Offset(WorldWind.OFFSET_FRACTION, 1, WorldWind.OFFSET_FRACTION, 0);
```

Comment:
@Beak-man This documentation is perfect.

With fresh eyes this morning, it occurred to me that the placement and the alignment seem backwards. This concerns code clarity, not correctness. What you have works, but I think others will struggle to understand how the code relates to the logo's appearance. Try explaining what negative inset pixels means, and you'll see what I'm getting at.

The following configuration is more clear:
```
this.logoPlacement = new Offset(WorldWind.OFFSET_INSET_PIXELS, 5, WorldWind.OFFSET_INSET_PIXELS, 36);
this.logoAlignment = new Offset(WorldWind.OFFSET_FRACTION, 1, WorldWind.OFFSET_FRACTION, 0);
```

Reply:
That looks good @Beak-man. The two different logo units could be an easily overlooked subtlety, but at least the meaning is clear. Plus, there's no better way to achieve this.

You're right, we don't need to match ScreenCreditController's configuration exactly. The 5 pixel margin is easy to understand and adjust as needed. Additionally, the positions of text attributions and this Bing logo will likely change in the next iteration anyway.

Reply:
It's done. There's another concern that I just noticed. The change of coordinates kind of reverses the meaning of "placement" and "alignment" if we go by the quantities. It now looks like `logoAlignment` is the one that locates the logo in the screen and `logoPlacement` provides the margins and alignment.